src/main/java/com/bloomtechlabs/fp/entities/EducationHistory.java

```diff
@@ -0,0 +1,88 @@
+package com.bloomtechlabs.fp.entities;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name ="education_histories")
+public class EducationHistory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
The database schema shows the id type as UUID instead of Long.

Reply:
It looks like java has a java.util.UUID that I can use. I'll ask if I can use this.